tools/tck-api/src/test/resources/org/opencypher/tools/tck/Foo.feature

```diff
@@ -39,6 +39,25 @@ Feature: Foo
       | 1 |
     And no side effects
 
+  Scenario: Reading a CSV
+    Given an empty graph
+    And there exists a CSV file with URL as $param, with rows:
+      | 1 | 'a' |
```

Comment:
I would test the whole roundtrip (multiple comments):

```suggestion
    And there exists a CSV file with URL as $param, with rows:
      | i | txt |
      | 1 | 'a' |
```

These data tables also expect headers, I think.